app/konnect/org-management/okta-role-mapping.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Map Konnect Roles to Okta Groups
+no_version: true
+badge: enterprise
+---
+By mapping Okta groups to [Konnect roles](/konnect/org-management/users-and-roles),
+you can manage user authorization directly through your Okta app.
```

Comment:
I would say "you can manage a user's Konnect roles directly through Okta group membership"